packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/plan-upgrade/template.html

```diff
@@ -0,0 +1,64 @@
+<div class="row mb-3">
+    <oui-select-picker
+        class="col-md-4 mb-3"
+        data-ng-repeat="plan in $ctrl.plans track by $index"
+        name="plan"
+        data-model="$ctrl.selectedPlan"
+        data-label="{{:: 'dedicatedCloud_plan_' + plan.offerName | translate }}"
+        data-variant="light"
+        data-on-change="$ctrl.onSelected(modelValue)"
+        data-values="[plan]"
+    >
+        <oui-select-picker-section>
+            <ul class="plan-list-style">
+                <li
+                    data-translate="dedicatedCloud_plan_upgrade_custom_report"
+                    data-ng-if="plan.customReport"
```

Comment:
this should come from BackupOffer object like offer.hasCustomReport(), make similar changes for other data-ng-if as well